src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -144,10 +143,6 @@ export class MeetingManager implements AudioVideoObserver {
   // Please use `meetingManagerConfig` to use `MeetingManagerConfig` values.
   simulcastEnabled: boolean = false;
```

Comment:
This is a `MeetingSessionConfiguration` property no?

Reply:
No :) 
https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html

Reply:
https://aws.github.io/amazon-chime-sdk-js/classes/meetingsessionconfiguration.html#enablesimulcastforunifiedplanchromiumbasedbrowsers ???

Reply:
Hmm, so this `simulcastEnabled` variable is used to enable two meetingSessionConfiguration properties i.e. 
```
configuration.enableUnifiedPlanForChromiumBasedBrowsers = true;
configuration.enableSimulcastForUnifiedPlanChromiumBasedBrowsers = true;
```
I thought meetingSessionConfiguration did not have `enableUnifiedPlanForChromiumBasedBrowsers` because that page is missing documentation. We probably should update that.

Reply:
I think we do not need this `simulcastEnabled` property then. I will go ahead and remove it.

Reply:
I have removed `simulcastEnabled` param from Meeting Provider. The builders will be able to enable simulcast by updating the `meetingSessionConfiguration` properties before passing to `meetingManager.join()`.